packages/effects-runtime/src/runtime.ts

```diff
@@ -22,7 +22,7 @@ export class UnhandledEffectError extends TypeError {
   constructor({ type }: Effect) {
     super();
 
-    this.message = `Encountered an unhandled effect :${String(type)}`;
```

Comment:
This spacing was intentional, `:<effect-name>` in an attempt to try to communicate that the effect type was sort of meta, like a Symbol.

Reply:
ya, i got the above error, and i thought "oh no, i've fat fingered an extra character somewhere" and then proceeded to search the codebase for `:childPerform`. `:childPerform` (obviously) didn't show up and i scratched my head. i then opened up my search for the error message prefix and saw that this statement was doing the thing. i get the appeal, and the desire for a cool ux here. it had the opposite initial effect for me, but maybe i was just being a dummy. i'll swap it back when i pick back up here. food for thought.

Reply:
It _may_ just be too cute. There's no realy rhyme or reason for this tag-style syntax... I say we remove if it was even the slightest confusing.

Reply:
e3c4cdf